pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -23,35 +23,42 @@
 with stdenv.lib;
 
 assert (!libsOnly) -> kernel != null;
```

Comment:
I mean, I didn't intend to be this nitpicky, but there parens are still unnecessary.

Reply:
To be exact, use
```nix
assert !libsOnly -> kernel != null
```